pages/user/installing-conda.md

```diff
@@ -0,0 +1,161 @@
+
+The steps of the installation procedure are (1) download an installer script, (2) run it, and (3) answer interactive prompts.
+
+Of the four combinations Miniforge gives you (`conda` vs `mamba`, Python vs PyPy), [we recommend mamba with Python, which is this table](https://github.com/conda-forge/miniforge#mambaforge). (Open that link in a new window.)
+
+Within each table is a list of architectures. On Mac and Linux, you can get the name of your architecture from
+
+```bash
+uname -i
+```
+
+It is very likely `x86_64`. Select the installation script for your architecture by clicking or right-clicking the link on the Miniforge page.
+
+On Mac or Linux, run the script with
+
+```bash
+bash filename-of-the-script-you-just-downloaded.sh
+```
+
+Windows has a `start` command; see [Miniforge's instructions](https://github.com/conda-forge/miniforge#windows).
+
+The interactive prompts will ask you where you want to install it (default is `~/mambaforge`) and whether you want to have it enabled whenever you start a new terminal or shell (probably "yes"). Saying "yes" to the latter inserts a "`>>> conda initialize`" section in your shell configuration (probably `~/.bashrc`).
+
+<details markdown="1"><summary>Deciding whether conda should take over your shell?</summary>
+
+If you say "yes" to let the installer script modify your shell configuration, then the next terminal you open will be in the conda environment. For instance,
+
+```bash
+python
+```
+
+will run the conda environment's Python, rather than any other Python you have installed on your computer. This is what conda calls the "base" environment (though you can create more environments that are independent of this one).
+
+If, instead, you want to explicitly opt-into conda environments by calling a command, use
+
+```bash
+conda config --set auto_activate_base false
+```
+
+to prevent the "base" environment from being automatically loaded in each new terminal. Now all environments, including "base", have to be explicitly activated with
+
+```bash
+conda activate name-of-environment
+```
+
+See [managing environments](https://docs.conda.io/projects/conda/en/latest/user-guide/getting-started.html#managing-environments) in the conda documentation for more.
+
+If you say "no" to not let the installer script modify your shell configuration, then you will have to manually find the path to the `conda` executable, which is in `~/mambaforge/bin/conda`. All of the above applies, but your shell might not be able to find `conda` or `python`.
+
+</details>
+
+## Regular maintenance of the environment
+
+Now you're ready to go. [Instructions online](https://docs.conda.io/projects/conda/en/latest/user-guide/cheatsheet.html) tell you how to install packages, like
+
+```bash
+conda install name-of-package
+```
+
+Since you installed `mamba`, you can replace `conda install` with `mamba install` to make the dependency resolution much faster.
```

Comment:
I only noticed this now: with `mambaforge` installed, does `conda install` and `mamba install` not do the exact same thing? I thought they're aliases of each other in this case. If that is correct, it may be confusing to compare the two commands here and talk about speed, as the user following instructions would not see a difference.

Reply:
No, conda is still the conda command. mamba does some of conda's subcommands, but not all. (And I think conda is actually a shell alias, while mamba is a straight command, so you can't `mamba activate`/`deactivate`). I could be wrong though.

Reply:
That's good to know, thanks!

Reply:
It is the case that you can't `mamba activate`/`deactivate`.